src/pages/check-answers/app.ts

```diff
@@ -22,19 +23,13 @@ export const lambdaHandler = withErrorHandling(async (event) => {
 const get = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
     try {
         const session = await getSession(event);
+        const otherPensionsSession = session['other-pension-providers'] || [];
+        const providers = otherPensionsSession.map((provider) => OtherPensionProvidersMap[provider]);
+
         return renderAsHtmlResponse(event, 'template.njk', {
             session,
             mapping: {
```

Comment:
I think it makes sense to do it this way rather than what I did before now we have multiple answers, but perhaps `mapping` needs renaming?